src/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
use styles for all modifiers from the stars task, so I can change the number in devtools and it will work